ratis-common/src/main/java/org/apache/ratis/util/LifeCycle.java

```diff
@@ -223,7 +223,7 @@ public State transitionAndGet(UnaryOperator<State> operator) {
    * If the current state is equal to the specified from state,
    * then transition to the give to state; otherwise, make no change.
    *
-   * @return true iff the current state is equal to the specified from state.
+   * @return true if the current state is equal to the specified from state.
```

Comment:
Thanks for checking the javadoc!

This is not a typo iff means "if and only if"; see https://www.merriam-webster.com/dictionary/iff

Reply:
Amazing,  I didn't know this usage before

Reply:
> This is not a typo iff means "if and only if";

Thanks for the info, I also not hear of this usage